101-webapp-basic-windows/azuredeploy.json

```diff
@@ -2,57 +2,87 @@
   "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "webAppName": {
-      "type": "string",
-      "metadata": {
-        "description": "Base name of the resource such as web app name and app service plan"
+      "webAppName": {
+          "type": "string",
+          "metadata": {
+              "description": "Web app name"
+          },
+          "minLength": 2
       },
-      "minLength": 2
-    },
-    "sku":{
-      "type": "string",
-      "defaultValue" : "S1",
-      "metadata": {
-        "description": "The SKU of App Service Plan, by default is Standard S1"
-      }
-    },
-    "location": {
-      "type": "string",
-      "defaultValue": "[resourceGroup().location]",
-      "metadata": {
-        "description": "Location for all resources"
+      "location": {
+          "type": "string",
+          "defaultValue": "[resourceGroup().location]",
+          "metadata": {
+              "description": "Location for all resources"
+          }
+      },
+      "sku": {
+          "type": "string",
+          "defaultValue": "F1",
+          "metadata": {
+              "description": "The SKU of App Service Plan "
+          }
+      },
+      "language": {
+          "type": "string",
+          "defaultValue": "default",
+          "metadata": {
+              "description": "The language stack of the app. Default = siteConfig. Java = javaSiteConfig. See objects below. "
+          }
+      },
+      "siteConfig": {
```

Comment:
move both site config params to a variable like in the example I sent... using params creates a few problems:
1) object params are not easy for humans to manipulate so the "hellow world" simplicity is lost
2) these params and become incompatible with the language param and that will cause deployment to fail